### ISSUE TITLE
"No Such File Or Directory"

### ISSUE THREAD

**Issue by chryton** [pgrimaud/action-shopify#4]
I am seeing an issue where the shopify action is not able to find the checked out theme. Below is a slim example of the action file:

```yml
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - uses: actions/checkout@v2

      - name: Deploy Shopify theme
        uses: pgrimaud/action-shopify@2.1.0
        env:
          SHOPIFY_PASSWORD: ${{ secrets.DEV_SHOPIFY_PASSWORD }}
          SHOPIFY_STORE_URL: ${{ secrets.DEV_SHOPIFY_STORE_URL }}
          SHOPIFY_THEME_ID: ${{ secrets.DEV_SHOPIFY_THEME_ID }}
          THEME_PATH: /github/workspace
```
**Error:** `readAsset: open /github/workspace/***: no such file or directory`

Regardless if I use `/github/workspace`, `./`, `$GITHUB_WORKSPACE`, or `${{ github.workspace }}` for the `THEME_PATH` it appears that the theme is in accessible by the themekit instance.

**Comment by pgrimaud** [comment]
Hello @chryton 

Weird... Did you try to set THEME_PATH value as env var?

Replace 

`THEME_PATH: /github/workspace` 

per 

`${{ secrets.THEME_PATH }}`

I guess env var should start by `secrets.`

Like this  : https://github.com/freyum/shopify-workflow-poc/blob/master/.github/workflows/production.yml#L22N

**Comment by chryton** [comment]
I initially tried it with as a secret. Using `./` returns this: 
`readAsset: open /github/workspace/***: no such file or directory`

Using `/github/workspace` returns this: 
`readAsset: open ***/***: no such file or directory`

It appears that docker is mapping the variable correctly (redacted some info):
```bash
/usr/bin/docker run --name ##HASH## --label ##LABEL## --workdir /github/workspace --rm -e SHOPIFY_PASSWORD -e SHOPIFY_STORE_URL -e SHOPIFY_THEME_ID -e THEME_PATH -e INPUT_ARGS -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/##REPO_NAME##/##REPO_NAME##":"/github/workspace"
```

**Comment by chryton** [comment]
An update on my end: I replicated your setup in my own private action (not with docker) and it works as expected. 

One thing I noticed that I don't think should matter but may make a difference is that when you are installing themekit you are calling `| python` instead of `| sudo python` in the themekit docs; I had to use sudo for my own action to work correctly. I'm not well versed in Docker so it may not be needed like it would be for a normal installation.
